cmds/core/cat/cat_test.go

```diff
@@ -59,3 +68,53 @@ func TestCat(t *testing.T) {
 		t.Fatalf("Reading files failed: got %v, want %v", b.Bytes(), someData)
 	}
 }
+
+func TestCatPipe(t *testing.T) {
+	var inputbuf bytes.Buffer
+	teststring := "testdata"
+	fmt.Fprintf(&inputbuf, "%s", teststring)
+	osStdin = &inputbuf
+
+	var out bytes.Buffer
+	osStdout = &out
+
+	emptyargs := make([]string, 0)
+	if err := cat(&out, emptyargs); err != nil {
+		t.Error(err)
+	}
+	if out.String() != teststring {
+		t.Errorf("Input string: %s and output: '%s' string are not equal", teststring, out.String())
+	}
+	resetGlobals()
+}
+
+func TestFileNotExist(t *testing.T) {
+	args := make([]string, 0)
+	filepath := "testdata/doesnotexist.txt"
+	args = append(args, filepath)
+	var b bytes.Buffer
+	if err := cat(&b, args); !errors.Is(err, os.ErrNotExist) {
+		t.Error(err)
+	}
+	t.Log(b.String())
```

Comment:
A bit more informative message here will be useful. Think of the person looking at a log of this in 5 years: what would be most helpful to them? You might even just remove line 99 entirely, since the string appearing in the log might be confusing.

Reply:
Whoops. This really shouldn't be here